FlexMeasures/flexmeasures
# Feature/add consultancy account

## Description

Add a Consultant account that has read access to accounts of clients. 

Use case: Our customer is the supplier to a bunch of other (client) accounts on the server. They should have read-only access to some accounts. At least this is the first step. So they can oversee what happens, for instance for servicing questions and problems.

## Look & Feel

The consultant will be able to see the accounts in their overview page and the assets they have access to. The consultant client will be able to see which account has read access to their account.

## How to test

Tests are added in flexmeasures/api/v3_0/tests/test_assets_api.py 
To test in the UI create a consultant account and a consultant client account in the CLI. Then go to the account pages and see the accounts represented.

## Further Improvements

Potential improvements to be done in the same PR or follow up Issues/Discussions/PRs.

## Related Items

The technical discussion can be found here: https://github.com/FlexMeasures/flexmeasures/discussions/868
The issue that this closes can be found here: #203

---
